test/lit/passes/remove-unused-brs-eh.wast

```diff
@@ -151,6 +172,8 @@
             (throw $e)
           )
         )
+        ;; Block the flow, so that the br above remains.
+        (unreachable)
```

Comment:
I modified some non-trivial tests to prevent this optimization from simplifying them further. But I let it get optimized in trivial ones where I don't think it affects readability.